pennylane/qchem/convert.py

```diff
@@ -602,12 +604,28 @@ def import_state(solver, tol=1e-15):
         wf_dict = _rccsd_state(solver, tol=tol)
     elif "UCCSD" in method:
         wf_dict = _uccsd_state(solver, tol=tol)
+    elif "tuple" in method:
+        if isinstance(solver[0][0], np.str_):
```

Comment:
The determinants must be a **list** (not an **array**) of integers for dmrg and a list of strings for shci.

Reply:
Copy that, changed that throughout.